### PR TITLE
Add BGE large en v1.5

### PR DESCRIPTION
**Pull request by isaac-chung**
Addresses part of https://github.com/embeddings-benchmark/arena/issues/6

## Checklist

- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`. 

### Adding a model checklist

 - [x] I have filled out the ModelMeta object to the extent possible
 - [x] I have ensured that my model can be loaded using
   - [x] `mteb.get_model(model_name, revision_id)` and
   - [x] `mteb.get_model_meta(model_name, revision_id)`
 - [x] I have tested the implementation works on a representative set of tasks.

### CODE REVIEW

File: mteb/models/jina_models.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from mteb.model_meta import ModelMeta
+
+paraphrase_multilingual_MiniLM_L12_v2 = ModelMeta(
+    name="jinaai/jina-embeddings-v2-base-en",
+    languages=["eng_Latn"],
+    open_source=True,
+    revision="f84569a8c478c7bb2ea3bc187034859870940deb",
+    release_date="2023-10-30",  # release date of paper
+)
```
Comment (Muennighoff) [inline mteb/models/jina_models.py:9]: Oh since Jina works out of the box with ST do we even need to add it? I guess it can already be loaded?

**Review by Muennighoff — COMMENTED** [review]
Amazing speed 🚀

File: mteb/models/jina_models.py (hunk 1 above)
Comment (isaac-chung) [reply mteb/models/jina_models.py]: Good call! Model loading worked without being added to `mteb/models/__init__.py`. Will remove

File: mteb/models/bge_models.py [hunk 2]
```diff
@@ -56,3 +56,12 @@ def encode_corpus(
...
 )
+
+bge_large_en_v1_5 = ModelMeta(
+    loader=partial(BGEWrapper, model_name="BAAI/bge-large-en-v1.5"),  # type: ignore
+    name="BAAI/bge-large-en-v1.5",
+    languages=["eng_Latn"],
+    open_source=True,
+    revision="d4aa6901d3a41ba39fb536a557fa166f842b0e09",
+    release_date="2023-09-12",  # initial commit of hf model.
+)
```
Comment (Muennighoff) [inline mteb/models/bge_models.py:67]: Don't we need `normalize_embeddings=True` when calling `self.mdl.encode`? 🤔
See
(1) https://huggingface.co/BAAI/bge-large-en-v1.5#using-sentence-transformers
(2) https://github.com/UKPLab/sentence-transformers/blob/2dee8c22f618df3f34f55d406bf97aa88383728a/sentence_transformers/SentenceTransformer.py#L430

Maybe worth checking on a quick retrieval dataset like SciFact if we get similar scores to what the model gets on the mteb leaderboard

**Review by KennethEnevoldsen — APPROVED** [review]

File: mteb/models/bge_models.py (hunk 2 above)
Comment (isaac-chung) [reply mteb/models/bge_models.py:71]: Good catch! I've run `mteb run -t SciFact -m BAAI/bge-large-en-v1.5 --model_revision d4aa6901d3a41ba39fb536a557fa166f842b0e09`. 

`bge-large-en-v1.5` gives 74.63 from mteb (this PR) and 74.61 from the leaderboard.

Comment (isaac-chung) [reply mteb/models/bge_models.py:75]: Rerunning with `normalize_embeddings=True` yields 74.63 as well.

**Review by Muennighoff — APPROVED** [review]

**Pull request merged**